internal/pkg/checks/defaultBackendCheck.go

```diff
@@ -0,0 +1,60 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package checks
+
+import (
+	"fmt"
+	"github.com/kubernetes-sigs/ingress-controller-conformance/internal/pkg/k8s"
+)
+
+func init() {
+	Checks.AddCheck(singleServiceCheck)
+}
+
+var singleServiceCheck = &Check{
+	Name:        "default-backend",
+	Description: "Ingress with a single default backend should send traffic to the correct backend service",
+	Run: func(check *Check, config Config) (success bool, err error) {
```

Comment:
This is not linked to this review, but why is success AND error both needed?

Reply:
The intent was to report the success/failure of the test assertions separately from the test execution error. This will probably change as https://github.com/kubernetes-sigs/ingress-controller-conformance/issues/18 is addressed.